addons/editor-compact/hide-labels.css

```diff
@@ -0,0 +1,63 @@
+/* Asset editor */
+[class*="asset-panel_detail-area"] [class*="labeled-icon-button_mod-edit-field"],
+[class*="sound-editor_tool-button_"],
+[class*="sound-editor_row-reverse_"] > [class*="icon-button_container_"] {
+  min-width: 0;
+  margin-right: 2px;
+  padding: calc(0.125rem - 1px);
+  border: 1px solid var(--editorDarkMode-border, hsla(0, 0%, 0%, 0.15));
+  border-radius: 0.25rem;
+}
+[class*="sound-editor_tool-button"] img {
+  width: 20px;
+}
+[class*="asset-panel_detail-area"] [class*="label_input-label"],
+[class*="asset-panel_detail-area"] [class*="labeled-icon-button_edit-field-title"],
+[class*="icon-button_title_"] {
+  display: none;
+}
+[dir="ltr"] [class*="paint-editor_mod-dashed-border_"],
+[dir="rtl"] [class*="paint-editor_mod-dashed-border_"] {
+  border: none;
+}
+[class*="mode-tools_mod-labeled-icon-height"],
+[class*="paint-editor_mod-labeled-icon-height"] {
+  height: auto;
+}
+[class*="paint-editor_row_"] > [class*="tools_row_"],
+[class*="paint-editor_mod-mode-tools_"] > [class*="mode-tools_mode-tools_"] {
+  gap: calc(0.25rem);
```

Comment:
@mxmou Have we been avoiding `gap` prior to this? It was rarely used in editor addon CSS before this.
Using it is perfectly fine, I'm just wondering if there's places where we have been avoiding it and it could result on cleaner CSS.